### DISCUSSION TITLE
Como obter a conquista de Quickdraw no GitHub?

### DISCUSSION THREAD

**Discussion by LucasHavranekValor** [community/community#55946] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Como é possível obter a conquista de QuickDraw no GitHub?

**Comment by LucasHavranek** [comment, marked as answer]
Para obter a conquista QuickDraw no GitHub é necessário resolver uma issue de uma repositório público ou privado em menos de 5 minutos após aberto a issue
